### DISCUSSION TITLE
Merge bookmarks

### DISCUSSION THREAD

**Discussion by dressito** [sepinf-inc/IPED#618] [Q&A]
Hello,

I'm wondering if there is some way to "merge cases"... I have two colleagues working on the same case (two copies in different computers). Each one has generated their own bookmarks, and now they want to know if it is possible to merge them in one single case with all the bookmarks.
I've tried to load the .iped file with the bookmarks, but this just replaces the bookmark set.

Thanks

**Comment by lfcnassif** [comment]
Currently not, but if you want to contribute, this could be implemented changing https://github.com/sepinf-inc/IPED/blob/master/iped-engine/src/main/java/dpf/sp/gpinf/indexer/search/Marcadores.java#L432 with a new param, so user could choose the desired behavior.

**Reply by dressito** [reply]
Thanks for the answer! I would like to contribute, but even though I know programming, I don't have a background in Java, so it would take me some time to get used to it. I do get the gist of the code, so I'll look into it if I can.
And for those that might run into the same issue, I've tried to open the cases (same case, different folder name) as a muticase, so it merges the bookmarks.
I thought that, maybe referencing the same evidence image it would "merge" them in the evidence tab, but it duplicates it, and all the files. But selecting the "filter duplicates" does the job.
Not the most elegant solution, but it is a functional workaround.

**Reply by lfcnassif** [reply]
Be careful, if the same file was put in different bookmarks in different case copies, I think after applying "filter duplicates" just one bookmark will be shown in bookmarks column for the specific file.

**Reply by dressito** [reply]
Makes sense.... I didn't test it personally, but I've told them, and they think that, as long as the file is listed, which bookmark remains is not important.
